From: Arnaud Ferraris <arnaud.ferraris@collabora.com>
Date: Wed, 29 Sep 2021 14:48:45 +0200
Subject: pine64_defconfig: reboot immediately on panic

Waking up from sleep sometimes causes the phone to hang, with no way to
recover it other than a hard reboot. This patch makes sure the kernel
reboots immediately so users aren't left with an inert phone.
---
 arch/arm64/configs/pine64_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/pine64_defconfig b/arch/arm64/configs/pine64_defconfig
index fc58224..72ad1e9 100644
--- a/arch/arm64/configs/pine64_defconfig
+++ b/arch/arm64/configs/pine64_defconfig
@@ -2395,6 +2395,7 @@ CONFIG_PAGE_POISONING=y
 CONFIG_SCHED_STACK_END_CHECK=y
 CONFIG_DEBUG_MEMORY_INIT=y
 CONFIG_MEMORY_NOTIFIER_ERROR_INJECT=m
+CONFIG_PANIC_TIMEOUT=-1
 CONFIG_SOFTLOCKUP_DETECTOR=y
 CONFIG_SCHEDSTATS=y
 CONFIG_BUG_ON_DATA_CORRUPTION=y
